crates/router/src/connector/adyen/transformers.rs

```diff
@@ -4087,19 +4071,41 @@ pub struct AdyenPayoutCancelRequest {
     merchant_account: Secret<String>,
 }
 
+#[cfg(feature = "payouts")]
+impl TryFrom<&PayoutMethodData> for PayoutCardDetails {
+    type Error = Error;
+    fn try_from(item: &PayoutMethodData) -> Result<Self, Self::Error> {
+        match item {
+            PayoutMethodData::Card(card) => Ok(Self {
+                _type: "scheme".to_string(), // FIXME: Remove hardcoding
+                number: card.card_number.peek().to_string(),
+                expiry_month: card.expiry_month.peek().to_string(),
+                expiry_year: card.expiry_year.peek().to_string(),
+                holder_name: card
+                    .card_holder_name
+                    .clone()
+                    .get_required_value("card_holder_name")
+                    .change_context(errors::ConnectorError::MissingRequiredField {
+                        field_name: "payout_method_data.card.holder_name",
+                    })?
+                    .peek()
+                    .to_string(),
```

Comment:
Is it okay for this to be mandatory? I don't have much context about payouts but for normal payments, the card holder name is not mandatory for a lot of connectors.

Reply:
For this API, this is a required parameter.